docs/core/tutorials/using-on-macos.md

```diff
@@ -220,4 +230,4 @@ Open the debugger tab by selecting the Debug icon in the Visual Studio Code tool
 Press the Play button to start the application under the debugger. The app begins execution and runs to the breakpoint, where it stops. Step into the `Get` method and make sure that you have passed in the correct arguments. Confirm that the answer is 42.
 
 <a name="dotnet-restore-note"></a>
-[!INCLUDE[DotNet Restore Note](~/includes/dotnet-restore-note.md)]
\ No newline at end of file
+[!INCLUDE[DotNet Restore Note](~/includes/dotnet-restore-note.md)]
```

Comment:
@ricardodsanchez why was this line changed?

Reply:
It wasn't intentional or manually changed. Not sure how/why this was updated.